scrapyd/webservice.py

```diff
@@ -35,7 +35,6 @@ def render_GET(self, txrequest):
 
         return {"node_name": self.root.nodename, "status":"ok", "pending": pending, "running": running, "finished": finished}
 
-
```

Comment:
This makes the style more consistent with the blank lines in the rest of the file
but is a counter-pep8 whitespace change that shouldn't be commited.
Perhaps I should fix all blank lines in a single commit.